docker-compose.yml

```diff
@@ -9,54 +11,53 @@ services:
         ports:
             - "9200:9200"
             - "9300:9300"
+
     postgres:
         image: postgres:10.3
         restart: always
         volumes:
-          - postgres_data:/var/lib/postgresql/data/
+            - postgres_data:/var/lib/postgresql/data/
         environment:
             POSTGRES_USER: cfpb
             POSTGRES_PASSWORD: cfpb
             POSTGRES_DB: cfgov
+
     python2:
-        build: 
-            context: ./
-            dockerfile: ./Dockerfile
-        image: python
+        build:
+            context: .
+            dockerfile: Dockerfile
+            target: cfgov-develop
+            args:
+                scl_python_version: python27
         ports:
```

Comment:
Any reason for not including the environment variables for connecting to postgres?

Reply:
I considered it, but since [`.env_SAMPLE` already has `DATABASE_URL`](https://github.com/cfpb/cfgov-refresh/blob/master/.env_SAMPLE#L65) defaulting to the Docker setup, and that file has to be sourced in the container anyway, _and_ it's used by non-Docker setups too, it felt like it'd add to confusion as to why just that one envvar was setup in compose.  In fact, that's why I removed the `ES_HOST` envvar from compose, since it was [duplicated in `.env_SAMPLE`](https://github.com/cfpb/cfgov-refresh/blob/master/.env_SAMPLE#L92) as well.